### ISSUE TITLE
.cc that checks comments

### ISSUE THREAD

**Issue by rossjrw** [rossjrw/tars#180]
Store time of last comment check.

Response: `You have [n] new comments on your articles since you checked [timeago]: [FILE]`

Attached text file containing each comment with timestamp/link/etc along with parent comment if relevant (perhaps only do this for large numbers of comments?)

Blocked by #173 and #170

**Comment by rossjrw** [comment]
As of #221, forums will not be propagated, and all issues associated with the checkcomments feature are closed as wontfix.
